lib/tasks/migration.rake

```diff
@@ -702,9 +714,38 @@ namespace :migration do
       puts "Summary: Save file time: #{save_time}"
       puts "Summary: Add to Collection time: #{collection_time}"
       puts "Summary: Verification time: #{verify_time}"
-
+      noid_report.close
   end
 
+    def update_handle(handle,noid)
+      link = "https://era.library.ualberta.ca/files/#{noid}"
+      handle.slice!("http://hdl.handle.net/")
+      MigrationLogger.info "Deleting existing handle #{handle}"
+      puts handle
+      delete_cmd = "./bin/handle/bin/hdl-delete 0.NA/10402 bin/handle/lib/admpriv.bin #{handle}"
```

Comment:
@pgwillia  It's been pulled out from the PR as it contains private authentication information. It's now installed on bournemouth. It probably makes more sense to use Ansible playbook to handle the installation. I will check with @henryzhang87